assets/css/styles.scss

```diff
@@ -1484,3 +1484,26 @@ form[name="restaurant"] {
     }
   }
 }
+
+.exclusive {
+
+  background:-webkit-gradient(linear, left top, left bottom, color-stop(0.05, #ffec64), color-stop(1, #ffab23));
```

Comment:
Since we are using [autoprefixer](https://github.com/postcss/autoprefixer), you don't need to add vendor prefixes.
Just keep the line with `background:linear-gradient(to bottom, #ffec64 5%, #ffab23 100%);`